docs/source/nested.rst

```diff
@@ -60,3 +60,10 @@ nested_tensor([
 ])
 >>> nt.unbind()
 [tensor([0., 1., 2.], device='cuda:0'), tensor([3., 4., 5., 6., 7.], device='cuda:0')]
+
```

Comment:
To match what's done with sparse, I suggest adding a `Nested tensor methods` section header here, with a description of `The following Tensor methods are related to nested tensors:` before the autosummary